release-notes/Cloud/DCP_change_log.md

```diff
@@ -9,6 +9,14 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+### 9 December 2024 - Enhancement - Home/Admin/Catalog/Marketplace/Sharing - Click on app title opens new tab of the app [ID 41524]
+
+From now on, you can open Admin, Catalog, Home, Marketplace, and Sharing in a new tab using middle-click or right-click options. This update makes their titles behave like regular links.
```

Comment:
```suggestion
### 9 December 2024 - Enhancement - Home/Admin/Catalog/Marketplace/Sharing - Right- or middle-clicking app title opens app in new tab [ID 41524]

When you click the title of the Admin, Catalog, Home, Marketplace, or Sharing apps with the right or middle mouse button, this will now open the app again in a new tab.
```